projects/UA_Kocot.yaml

```
@@ -0,0 +1,5 @@
+Description: Biodiversity and evolution of marine invertebrates
+Department: Department of Biological Sciences
+FieldOfScience: Biological and Biomedical Sciences
+Organization: The University of Alabama
```

```suggestion
Organization: University of Alabama
```
This is the organization that's used in the other UA projects.